<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select image</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileNameTextbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>11</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cutSizeEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="plainText">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="overlapSizeEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="plainText">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlap size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start cut</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputFilenameEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Helvetica Neue DeskInterface'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;finalImage.jpg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Leeway</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="leewayEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="MouseQGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>500</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MouseQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mouseqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
